third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -22,6 +24,6 @@ pub struct Experiment();
 
 impl Experiment {
     /// A flag which allows to turn off safety checks, like reference safety.
-    /// Rentention: permanent.
+    /// Retention: permanent.
```

Comment:
Interesting, I would have expected this to be temporary. What is the reasoning behind making this permanent?